lib/middleware/error_handler.coffee

```diff
@@ -7,6 +7,7 @@ module.exports = (err, req, res, next) ->
     '../../desktop/components/error_handler/index.jade'
   )
   isNotProduction = NODE_ENV isnt 'production'
+  console.log err if isNotProduction
```

Comment:
Fixes issue where stylus compile-time errors don't report in console and dev has no clue whats going on